global.css

```diff
@@ -68,13 +68,7 @@
 	}
 }
 
-.hljs {
-	@apply my-4 w-full rounded-lg bg-neutral-100 text-sm dark:bg-neutral-900 !important;
-	/* Fixes overflowing code blocks */
-	white-space: break-spaces;
-	line-break: anywhere;
-	font-family: 'Courier New', Courier, monospace;
-}
+
 .discord-message a {
```

Comment:
Probs should be removed and moved into the parser

Reply:
Will do 👍